### PR TITLE
lang detection: honor app's AppleLanguages if available

### PR DESCRIPTION
**Pull request by zprd**
- on MacOS it is possible to set default languages per domain/app
    like: defaults write org.grisbi.Grisbi AppleLanguages "(fr, en)"

- check if App's languages are set if not use Global defaults

### CODE REVIEW

**Comment by LudovicRousseau** [comment]
Looks fine.
I will review it in a few days.

**Comment by LudovicRousseau** [comment]
It looks like the `/dev/null`redirection does not work. I still get some output:
```
$ defaults read org.grisbi.Grisbi AppleLanguages > /dev/null
2019-07-29 23:34:26.911 defaults[6010:68410] 
The domain/default pair of (org.grisbi.Grisbi, AppleLanguages) does not exist
```

Maybe you want to use `&>` instead of `>`?

File: MacOS/Grisbi.sh [hunk 1]
```diff
@@ -50,7 +50,14 @@ unset LANG LC_MESSAGES LC_MONETARY LC_COLLATE
 # Has a language ordering been set?
 # If so, set LC_MESSAGES and LANG accordingly; otherwise skip it.
 # First step uses sed to clean off the quotes and commas, to change - to _, and change the names for the chinese scripts from "Hans" to CN and "Hant" to TW.
-APPLELANGUAGES=`defaults read .GlobalPreferences AppleLanguages | sed -En   -e 's/\-/_/' -e 's/Hant/TW/' -e 's/Hans/CN/' -e 's/[[:space:]]*\"?([[:alnum:]_]+)\"?,?/\1/p' `
+if `defaults read org.grisbi.Grisbi AppleLanguages > /dev/null`; then
```
Comment (zprd) [inline MacOS/Grisbi.sh]:
```suggestion
if `defaults read org.grisbi.Grisbi AppleLanguages > /dev/null 2>&1`; then
```

**Comment by LudovicRousseau** [comment]
Fixed in https://github.com/grisbi/grisbi/commit/ccc79533c1376003d72bc308ad3fe010e1de1e12

Thanks

**Pull request closed without merging**